From c9c1e4d58aa598440904280e3261760fa6f0864e Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@nxp.com>
Date: Fri, 19 May 2017 11:39:25 +0300
Subject: [PATCH] config: add SDK QBMan driver

Signed-off-by: Madalin Bucur <madalin.bucur@nxp.com>
[ Aisheng: fix minor conflict due to removed CONFIG_VBOXSF_FS ]
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 drivers/staging/Makefile | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -53,3 +53,4 @@ obj-$(CONFIG_UWB)		+= uwb/
 obj-$(CONFIG_USB_WUSB)		+= wusbcore/
 obj-$(CONFIG_EXFAT_FS)		+= exfat/
 obj-$(CONFIG_QLGE)		+= qlge/
+obj-$(CONFIG_FSL_SDK_DPA)      += fsl_qbman/
